fitlins/workflows/base.py

```diff
@@ -1,12 +1,22 @@
 from pathlib import Path
 import warnings
-
+from nipype.pipeline import engine as pe
+from nipype.interfaces import utility as niu
+# from nipype.interfaces import fsl
+from ..interfaces.bids import (
+    ModelSpecLoader, LoadBIDSModel, BIDSSelect, BIDSDataSink)
+from ..interfaces.nistats import FirstLevelModel, SecondLevelModel
+from ..interfaces.visualizations import (
+    DesignPlot, DesignCorrelationPlot, ContrastMatrixPlot, GlassBrainPlot)
+from ..interfaces.utils import MergeAll, CollateWithMetadata
+from ..interfaces.afni import AFNIMergeAll 
```

Comment:
These are now being imported inside `init_fitlins_wf`. This works around some annoying new nipype behavior.

Reply:
good to know

Reply:
This was what was causing the packaging test failure.

Reply:
Aha. That confused me.